### ISSUE TITLE
character maps to undefined

### ISSUE THREAD

**Issue by User-Ali** [qxcv/caffe2keras#3]
`D:\caffe2keras-master\caffe2keras-master>python -m caffe2keras
network/TrainedModel.prototxt network/TrainedModel.caffemodel DeepNat_model.h5
Using TensorFlow backend.
Converting model...
Traceback (most recent call last):
  File "C:\Users\User\Anaconda3\envs\tensorflow\lib\runpy.py", line 193, i
n _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\User\Anaconda3\envs\tensorflow\lib\runpy.py", line 85, in
 _run_code
    exec(code, run_globals)
  File "D:\caffe2keras-master\caffe2keras-master\caffe2keras\__main__.py", l
ine 32, in <module>
    main()
  File "D:\caffe2keras-master\caffe2keras-master\caffe2keras\__main__.py", l
ine 22, in main
    model = convert.caffe_to_keras(args.prototxt, args.caffemodel, args.debug)
  File "D:\caffe2keras-master\caffe2keras-master\caffe2keras\convert.py", li
ne 353, in caffe_to_keras
    prototext = preprocess_prototxt(prototext)
  File "D:\caffe2keras-master\caffe2keras-master\caffe2keras\convert.py", li
ne 389, in preprocess_prototxt
    p = fp.readlines()
  File "C:\Users\User\Anaconda3\envs\tensorflow\lib\encodings\cp1252.py",
line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 30: chara
cter maps to undefined`

Can anyone please help me out ?

**Comment by qxcv** [comment]
It looks like it's trying to open a `.prototxt` with your operating system's default text encoding (CP1252), but the file has been saved in some other encoding (I assume UTF-8). The ideal fix would be to change the code to figure out text encoding automatically, but if you Just Want It To Work(tm), then the easiest fix is to open the `.prototxt` in your favourite text editor and save it in the CP1252 character set. How to do this depends on your text editor; Googling how to convert to CP1252 should bring up instructions.

**Comment by User-Ali** [comment]
Thanks ! It did solve my above problem. 

How stupid I am to not  notice that Windows had changed some characters (to be precise those open and close quotes (not " "); to see an example, look at the toolbar of comment) to some weird characters. 

I would like to tell here that initially, I just did change the encoding and it did solve that error but I got a new error which was `looking for strings, but found "` . So, I replaced all those quotes with `" "` and it worked. 

Thanks again.
